src/Resources/views/admin/activities/list.html.twig

```diff
@@ -57,7 +57,7 @@
                                         {{ 'cmf_activities_details' | trans({}, 'admin') }}
                                     </a>
                                 </td>
-                                <td width="160">{{ activity.activityDate.formatLocalized('%x %X') }}</td>
+                                <td width="160">{{ activity.activityDate.isoFormat('L LTS') }}</td>
```

Comment:
something like this?
```suggestion
                                {# This will display date and time based on locale. The format are moment.js parameters #}
                                <td width="160">{{ activity.activityDate.isoFormat('L LTS') }}</td>
```

personally i don't find it necessary, it's hardcoded and wouldn't expect anybody to change this in the template. 
Maybe need to move this conversion to the backend/php.

Reply:
that is true, it is not really necessary, but it does not hurt either. I was just not sure myself, that's why I looked it up in the first place :)